journal/player.hpp

```diff
@@ -8,6 +8,8 @@
 
 #include "serialization/journal.pb.h"
 
+#define PRINCIPIA_PLAYER_ALLOW_VERSION_MISMATCH 1
```

Comment:
Maybe default to 0 so you notice next time?

Reply:
I set it to 1 to check that it was properly disabling the check and forgot to flip back.  Sigh.